includes/data-port/class-sensei-data-port-utilities.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * File containing the Sensei_Data_Port_Utilities.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * A collection of utilies used in data port.
+ */
+class Sensei_Data_Port_Utilities {
+
+	/**
+	 * Create or update an existing user. If the user exists and the email is different, it will be updated.
+	 *
+	 * @param string $username  The username.
+	 * @param string $email     User's email.
+	 *
+	 * @return int|WP_Error
+	 */
+	public static function create_or_update_user( $username, $email = '' ) {
+		$user = get_user_by( 'login', $username );
+
+		if ( ! $user ) {
+			return wp_create_user( $username, $email, wp_generate_password() );
+		}
+
+		if ( $user->user_email !== $email && ! empty( $email ) ) {
```

Comment:
I don't think we should update the user email if it doesn't match.

Reply:
I was just thinking that in case the user makes a typo on the email, then we don't offer them a way to quickly correct it. On the other side this error might not be that common and manually correcting the email is probably easy enough. 7ecd3c212d2769b7368100762355c39ed2b9fa78